tests/testthat/test-filter.R

```diff
@@ -0,0 +1,229 @@
+# TODO: implement these functions in filters.R and use them in this test
+include <- function(...) {}
+exclude <- function(...) {}
+lessthan <- function(...) {}
+morethan <- function(...) {}
+interval <- function(...) {}
+
+data(original_data)
+original_samples_count <- nrow(original_data$data)
+
+test_cases <- list()
+
+test_cases[[length(test_cases) + 1]] <- list(
+  data_factory = function() {
+    data(test_data)
```

Comment:
you don't need to import test_data in each function; you can do it once in global env and on each call function will use unchanged test_data from global env.